framework/docker/dataavailability/network.go

```diff
@@ -49,3 +73,82 @@ func (n *Network) GetFullNodes() []*Node {
 func (n *Network) GetLightNodes() []*Node {
 	return n.GetNodesByType(types.LightNode)
 }
+
+// AddNodes adds one or more nodes to the DA network with the given configurations.
+// The nodes are created and initialized but not started - call Start() on the returned nodes to start them.
+func (n *Network) AddNodes(ctx context.Context, nodeConfigs ...NodeConfig) ([]*Node, error) {
```

Comment:
this works in a similar way to the AddNode function on the `Chain`, I just made this one take a var args list of NodeConfigs so you can create multiple in parallel. We can maybe do the same for `AddChain` in a future PR.